pom.xml

```diff
@@ -1097,6 +1097,7 @@
                             <goal>clean</goal>
                         </goals>
                         <configuration>
+                            <excludeDefaultDirectories>true</excludeDefaultDirectories>
```

Comment:
Leaving `target` folder as-is is dangerous - when I clean something I want it to be clean.